### PR TITLE
Add owned resources API markers

### PR DESCRIPTION
**Pull request by andrewazores**
Related to #150

This fixes the scorecard test `olm-crds-have-resources`.

### CODE REVIEW

File: api/v1beta1/flightrecorder_types.go [hunk 1]
```diff
@@ -148,6 +148,7 @@ type JMXAuthSecret struct {
 // +kubebuilder:subresource:status
 
 // FlightRecorder is the Schema for the flightrecorders API
+//+operator-sdk:csv:customresourcedefinitions:resources={{Recording,v1beta1},{Secret,v1},{Service,v1}}
```
Comment (ebaron) [inline api/v1beta1/flightrecorder_types.go:151]: This one doesn't actually use Recording, but you could add `{Pod,v1}`.

File: api/v1beta1/recording_types.go [hunk 2]
```diff
@@ -112,6 +112,7 @@ type RecordingStatus struct {
 // +kubebuilder:resource:path=recordings,scope=Namespaced
 
 // Recording is the Schema for the recordings API
+//+operator-sdk:csv:customresourcedefinitions:resources={{Pod,v1beta1},{Secret,v1},{Service,v1}}
```
Comment (ebaron) [inline api/v1beta1/recording_types.go:115]: Pod should be `v1`.

Comment (andrewazores) [reply api/v1beta1/recording_types.go:115]: Whoops, I had that as `Recording` initially as we discussed on irc. Glad you caught it.

File: api/v1beta1/containerjfr_types.go [hunk 3]
```diff
@@ -93,6 +93,7 @@ type PersistentVolumeClaimConfig struct {
 // +kubebuilder:resource:path=containerjfrs,scope=Namespaced
 
 // ContainerJFR is the Schema for the containerjfrs API
+//+operator-sdk:csv:customresourcedefinitions:resources={{Deployment,v1},{PersistentVolumeClaim,v1},{Secret,v1},{Service,v1},{Route,v1},{FlightRecorder,v1beta1},{ConsoleLink,v1}}
```
Comment (ebaron) [inline api/v1beta1/containerjfr_types.go:96]: Maybe remove FlightRecorder here, but add `{Ingress,v1}`? This is coming with #160

**Comment by ebaron** [comment]
Looks good, just needs a rebase.

**Review by ebaron — APPROVED** [review]

**Pull request merged**